eras/conway/impl/src/Cardano/Ledger/Conway/TxBody.hs

```diff
@@ -491,8 +502,10 @@ encodeTxBodyRaw ::
 encodeTxBodyRaw ConwayTxBodyRaw {..} =
   let ValidityInterval bot top = ctbrVldt
    in Keyed
-        ( \i ifee ri o cr tc f t c w b rsh mi sh ah ni ga ->
-            ConwayTxBodyRaw i ifee ri o cr tc c w f (ValidityInterval b t) rsh mi sh ah ni ga
+        ( \i ifee ri o cr tc f t c w b rsh mi sh ah ni ga ctv ->
+            -- QUESTION: Why is there `ga` when that has already been split into
+            -- `VotingProcedures` and `ProposalProcedures`? Should it not be ` vp pp`?
+            ConwayTxBodyRaw i ifee ri o cr tc c w f (ValidityInterval b t) rsh mi sh ah ni ga ctv
```

Comment:
Answer: Because it is eta reduced. So, in the current code above `ga` corresponds to `votingProcedures`, `ctv` corresponds to `proposalProcedures` and the new field you added is eta reduced. So, to avoid confusion and potential mistakes lets eta reduce as much as possible:
```suggestion
        ( \i ci ri o cr tc f t c w b ->
            ConwayTxBodyRaw i ci ri o cr tc c w f (ValidityInterval b t)
```

Reply:
Got it! Thanks! :raised_hands: